frontends/ol-components/src/components/Carousel/Carousel.tsx

```diff
@@ -103,24 +102,24 @@ const Carousel: React.FC<CarouselProps> = ({
         {children}
       </NukaCarouselStyled>
       <Stack direction="row" justifyContent="end" spacing={3} marginTop={3}>
-        <FilledIconButton
-          variant="contained"
+        <ActionButton
           size="small"
+          edge="rounded"
           onClick={pageDown}
           disabled={!canPageDown}
           aria-label="Previous"
         >
-          <NavigateBeforeIcon fontSize="small" />
-        </FilledIconButton>
-        <FilledIconButton
-          variant="contained"
+          <RiArrowLeftLine />
+        </ActionButton>
+        <ActionButton
           size="small"
+          edge="rounded"
           onClick={pageUp}
           disabled={!canPageUp}
           aria-label="Next"
         >
-          <NavigateNextIcon fontSize="small" />
-        </FilledIconButton>
+          <RiArrowRightLine />
```

Comment:
I sapped the icons. Carousels still do not match designs, but at least icon is right now.